src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -956,6 +956,10 @@ public function setNotModified()
         $this->setStatusCode(304);
         $this->setContent(null);
 
+        if (property_exists($this, 'streamed')) {
+            $this->streamed = true;
+        }
+
```

Comment:
Added a comment below with more information about how I'm using the streamedResponse that causes this issue to happen.